<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KEmuWindow</class>
 <widget class="QMainWindow" name="KEmuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KEmu</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-engineering">
    <normaloff/>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="startStopButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="icon">
       <iconset theme="system-run">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" rowspan="2" colspan="3">
     <widget class="KEditListWidget" name="machinesList">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>CD-ROM image</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KUrlRequester" name="CDROMInput"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Hard Disk image</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="systemComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>System type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KUrlRequester" name="HardDiskInput"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Video output</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="videoComboBox">
         <item>
          <property name="text">
           <string>cirrus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vmware</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>qxl</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>xenfb</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>tcx</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cg3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>virtio</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Random Access Memory</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="KIntNumInput" name="RAMInput">
         <property name="minimum">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>471</width>
           <height>373</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="KVMCheckBox">
         <property name="text">
          <string>Kernel-based Virtual Machine </string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="KLineEdit" name="argumentsLineEdit"/>
       </item>
       <item row="6" column="1">
        <widget class="KIntNumInput" name="CPUInput">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Extra arguments</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>CPU cores</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="ACPICheckBox">
         <property name="text">
          <string>Advanced Configuration and Power Interface</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="audioComboBox">
         <item>
          <property name="text">
           <string>sb16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>es1370</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ac97</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>adlib</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cs4231a</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>hda</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pcspk</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>all</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Audio output</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
